spec/schemas/nodes.info.yaml

```diff
@@ -21,21 +21,24 @@ components:
         - aggregations
         - indices
         - search_pipelines
+        - _all
+        - _cluster_manager
     ResponseBase:
       allOf:
         - type: object
           properties:
             cluster_name:
               $ref: '_common.yaml#/components/schemas/Name'
             nodes:
-              type: object
-              additionalProperties:
-                $ref: '#/components/schemas/NodeInfo'
+              oneOf:
+                - type: object
+                  $ref: '#/components/schemas/NodeInfo'
+                - type: object
+                  additionalProperties: {}
             _nodes:
                 $ref: 'nodes._common.yaml#/components/schemas/NodesResponseBase'
           required:
             - cluster_name
-            - nodes
             - _nodes
```

Comment:
These changes are likely due to the duplicate path and having the wrong schemas, please revert them and test again with other changes made.